### PR TITLE
commit poc

### PR DESCRIPTION
**Pull request by tyzbit**

### CODE REVIEW

**Comment by tyzbit** [comment]
terraform plan

**Comment by tyzbit** [comment]
terraform plan

**Comment by tyzbit** [comment]
terraform plan

**Comment by tyzbit** [comment]
terraform plan

**Comment by tyzbit** [comment]
atlantis help

**Comment by tyzbit** [comment]
atlantis plan

File: newrelic/local.tf [hunk 1]
```diff
@@ -0,0 +1,129 @@
+data "newrelic_synthetics_monitor" "torrent" {
```
Comment (Dirrk) [inline newrelic/local.tf:1]: This isn't needed

File: newrelic/local.tf [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+}
+
+resource "newrelic_synthetics_alert_condition" "torrent" {
+  policy_id = newrelic_alert_policy.web-checks.id
+
+  name       = "Torrent Web Alert Policy"
+  monitor_id = data.newrelic_synthetics_monitor.torrent.id
+}
+
+data "newrelic_synthetics_monitor" "rancher" {
```
Comment (Dirrk) [inline newrelic/local.tf:27]: same here, its not needed

File: newrelic/local.tf [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+  uri               = "https://torrent.qtosw.com"
+  validation_string = "WebUI"
+  verify_ssl        = true
+}
+
+resource "newrelic_synthetics_alert_condition" "torrent" {
+  policy_id = newrelic_alert_policy.web-checks.id
+
+  name       = "Torrent Web Alert Policy"
+  monitor_id = data.newrelic_synthetics_monitor.torrent.id
```
Comment (Dirrk) [inline newrelic/local.tf:24]: `newrelic_synthetics_monitor.torrent.id`

File: newrelic/local.tf [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+  uri               = "https://rancher.qtosw.com"
+  validation_string = "Loading"
+  verify_ssl        = true
+}
+
+resource "newrelic_synthetics_alert_condition" "rancher" {
+  policy_id = newrelic_alert_policy.web-checks.id
+
+  name       = "Rancher Web Alert Policy"
+  monitor_id = data.newrelic_synthetics_monitor.rancher.id
```
Comment (Dirrk) [inline newrelic/local.tf:50]: `newrelic_synthetics_monitor.rancher.id`

**Review by Dirrk — CHANGES_REQUESTED** [review]

**Pull request merged**